app/services/usps_in_person_proofing/mock/fixtures.rb

```diff
@@ -115,10 +115,6 @@ def self.request_in_progress_proofing_results_response
         load_response_fixture('request_in_progress_proofing_results_response.json')
       end
 
-      def self.request_enrollment_code_response
-        load_response_fixture('request_enrollment_code_response.json')
```

Comment:
Should the file `app/services/usps_in_person_proofing/mock/responses/request_enrollment_code_response.json` be removed as well?

Reply:
See commit [210c9e0](https://github.com/18F/identity-idp/pull/11049/commits/210c9e09f9396b04d162a20f7122210339ed20ee).